docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,217 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+The `ALTER VIEW` statement can alter metadata associated with the view. It can change the definition of the view, change
+the name of a view to a different name, set and unset the metadata of the view by setting `TBLPROPERTIES`.
+
+#### Rename VIEW
```

Comment:
VIEW isn't a keyword. RENAME is. I'd consistently all-caps the keywords, or not. Could be "RENAME View" or just "Rename View", but pick consistently.

Reply:
ok